source/manual/republishing-content.html.md.erb

```diff
@@ -16,17 +16,23 @@ You may wish to test first on integration, prior to carrying out the republish i
 
 ## Whitehall
 
-If the documents are in Whitehall, there are Rake tasks you can run as outlined below. Try to pick the one most focused to the scope of what you need to republish to avoid unnecessary load. You can monitor the effect on the publishing queue via these Grafana dashboards:
+If the documents are in Whitehall, you can republish content either via a user interface or a Rake task, as outlined below. Try to pick the task most focused to the scope of what you need to republish to avoid unnecessary load. You can monitor the effect on the publishing queue via these Grafana dashboards:
 
 - [integration](https://grafana.integration.publishing.service.gov.uk/dashboard/file/sidekiq.json?refresh=1m&orgId=1&var-Application=whitehall&var-Interval=$__auto_interval)
 - [staging](https://grafana.blue.staging.govuk.digital/dashboard/file/sidekiq.json?refresh=1m&orgId=1&var-Application=whitehall&var-Interval=$__auto_interval)
 - [production](https://grafana.blue.production.govuk.digital/dashboard/file/sidekiq.json?refresh=1m&orgId=1&var-Application=whitehall&var-Interval=$__auto_interval)
 
-### To republish a single document by slug
+### Individual content
 
-<%= RunRakeTask.links("whitehall-admin", "publishing_api:republish:document_by_slug[slug]") %>
+You can republish individual pieces of content - documents, organisations,
+people, roles, and a few specific pages (corresponding to custom presenters) -
+via a user interface. A link to this can be found in the "More" section of the
+site.
```

Comment:
Would it be possible to include a link to this section, for each of the three environments?